server/services/participants/participants-multi.ts

```diff
@@ -0,0 +1,288 @@
+import readXlsxFile from 'node-xlsx';
```

Comment:
I don't think I love the `participant-single` and `participants-multi` distinction. I think separating out the batch update stuff may be the way to go, and `getParticipants` can be lumped in with the stuff in `-single`.

Reply:
Good idea, changed as suggested.